packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -66,97 +68,141 @@ export class ListboxOption extends FoundationElement {
     @attr({ mode: "boolean" })
     public disabled: boolean;
     protected disabledChanged(prev, next): void {
-        if (this.proxy instanceof HTMLOptionElement) {
+        if (this.hasProxy) {
             this.proxy.disabled = this.disabled;
         }
     }
 
+    public get form(): HTMLFormElement | null {
+        return this.proxy?.form;
+    }
+
+    private get hasProxy() {
+        return this.proxy instanceof HTMLOptionElement;
+    }
+
     /**
-     * The selected attribute value. This sets the initial selected value.
+     * The initial value of the option. This value sets the `value` property
+     * only when the `value` property has not been explicitly set.
      *
-     * @public
      * @remarks
-     * HTML Attribute: selected
+     * HTML Attribute: value
      */
-    @attr({ attribute: "selected", mode: "boolean" })
-    public selectedAttribute: boolean;
-    protected selectedAttributeChanged(): void {
-        this.defaultSelected = this.selectedAttribute;
-
-        if (this.proxy instanceof HTMLOptionElement) {
-            this.proxy.defaultSelected = this.defaultSelected;
+    @attr({ attribute: "value", mode: "fromView" })
+    public initialValue: string;
+    public initialValueChanged(previous: string, next: string): void {
+        // If the value is clean and the component is connected to the DOM
+        // then set value equal to the attribute value.
+        if (!this.dirtyValue) {
+            this.value = this.initialValue;
+            this.dirtyValue = false;
         }
     }
 
+    /**
+     * Returns the value or text content
+     * @public
+     */
+    public get label() {
+        return this.value ?? this.textContent ?? "";
+    }
+
+    /**
+     * @internal
+     */
+    private parentListbox: Listbox | null;
+
+    /**
+     * @internal
+     */
+    public proxy: HTMLOptionElement;
+
     /**
      * The checked state of the control.
      *
      * @public
      */
     @observable
     public selected: boolean = this.defaultSelected;
-    protected selectedChanged(): void {
-        if (this.$fastController.isConnected) {
-            if (!this.dirtySelected) {
-                this.dirtySelected = true;
-            }
+    protected selectedChanged(prev: unknown, next: boolean): void {
+        if (!this.dirtySelected) {
+            this.dirtySelected = true;
+        }
```

Comment:
Nit: this can be just this.dirtySelected = true since it only does it when it's not true.